src/phew/server.py

```diff
@@ -251,41 +110,6 @@ async def _parse_json_body(reader, headers):
     return body_str, json.loads(body_str)
 
 
-status_message_map = {
-    200: "OK",
-    201: "Created",
-    202: "Accepted",
-    203: "Non-Authoritative Information",
-    204: "No Content",
-    205: "Reset Content",
-    206: "Partial Content",
-    300: "Multiple Choices",
-    301: "Moved Permanently",
-    302: "Found",
-    303: "See Other",
-    304: "Not Modified",
-    305: "Use Proxy",
-    306: "Switch Proxy",
-    307: "Temporary Redirect",
-    308: "Permanent Redirect",
-    400: "Bad Request",
-    401: "Unauthorized",
-    403: "Forbidden",
-    404: "Not Found",
-    405: "Method Not Allowed",
-    406: "Not Acceptable",
-    408: "Request Timeout",
-    409: "Conflict",
-    410: "Gone",
-    414: "URI Too Long",
-    415: "Unsupported Media Type",
-    416: "Range Not Satisfiable",
-    418: "I'm a teapot",
-    500: "Internal Server Error",
-    501: "Not Implemented",
-}
```

Comment:
instead of giving nice error messages I just return the same number as the message.